recipes/top2vec/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "top2vec" %}
+{% set version = "1.0.26" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/top2vec-{{ version }}.tar.gz
+  sha256: cd6ffc093b6ac2e24140df91e68b7332be013a969815d75714ebd06b3a022de7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python >=3.6
+    - cython
```

Comment:
Are you sure `cython` and a build section is required? Upstream looks like pure normal python.

Reply:
In that case please open an issue with `hdbscan` as that seems like an error with them: https://github.com/conda-forge/hdbscan-feedstock

If they don't reply soon we can keep it in here for now but then it would only be needed under `run`.